addon/index.js

```diff
@@ -4,6 +4,7 @@ import objectToArray from 'ember-changeset/utils/computed/object-to-array';
 import isEmptyObject from 'ember-changeset/utils/computed/is-empty-object';
 import isPromise from 'ember-changeset/utils/is-promise';
 import isObject from 'ember-changeset/utils/is-object';
+import recursiveKeys from 'ember-changeset/utils/recursive-keys';
```

Comment:
Let's change the name of this util to `leafKeys`, since this util excludes intermediary keys.